charts/elemental/103.1.0+up1.4.2/values.yaml

```diff
@@ -0,0 +1,43 @@
+image:
+  empty: rancher/pause:3.1
+  repository: "rancher/mirrored-elemental-operator"
+  tag: "1.4.2"
+  imagePullPolicy: IfNotPresent
+
+seedImage:
+  repository: "rancher/mirrored-elemental-seedimage-builder"
+  tag: "1.4.2"
+  imagePullPolicy: IfNotPresent
+
+channel:
+  image: "registry.suse.com/rancher/elemental-channel"
```

Comment:
note: changed the variable from "repository" to "image" to avoid Rancher scripts to save the image for the airgap